### PR TITLE
Bump to v0.18.1

### PR DESCRIPTION
**Pull request by ZCMax**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
Bump to v0.18.0.

## Modification
Please briefly describe what modification is made in this PR.

Update README.md
Update README_zh-CN.md
Update docs/en/getting_started.md
Update docs/zh_cn/getting_started.md
Update docs/en/changelog.md
Update mmdet3d/version.py

## BC-breaking (Optional)
Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)
If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/en/changelog.md [hunk 1]
```diff
@@ -1,6 +1,29 @@
...
-### v0.18.0 (1/1/2021)
+### v0.18.1 (1/2/2022)
+
+#### Improvements
+
+- Support Flip3D augmentation in semantic segmentation task (#1182)
+- Add point cloud annotation tools introduction in FAQ (#1185)
+- Add missing explanations of `cam_intrinsic` in the nuScenes dataset doc (#1193)
+- Update regnet metafile (#1184)
+- Add OpenMMLab website and platform links (#1176)
```
Comment (Tai-Wang) [inline docs/en/changelog.md:11]: I would prefer sorting the items according to their PR numbers (ascendingly).

Comment (Tai-Wang) [reply docs/en/changelog.md:11]: #1176 can be ignored

File: docs/en/changelog.md [hunk 2]
```diff
@@ -1,6 +1,29 @@
...
+
+#### Bug Fixes
+
+- Fix corner generation bug for empty bboxes in three coordinates (#1212)
```
Comment (Tai-Wang) [inline docs/en/changelog.md:15]: Fix a potential bug when generating corners for empty bounding boxes

File: docs/en/changelog.md [hunk 3]
```diff
@@ -1,6 +1,29 @@
...
+- Fix bug in non-distributed multi-gpu training/testing (#1197)
+- Fix rotation matrix while rotation axis=0 (#1182)
```
Comment (Tai-Wang) [inline docs/en/changelog.md:17]: Check the use of a/the for all descriptions

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**